hack/lib/mesh_resources/authorization-policies/helm/eventing-e2e0.yaml

```diff
@@ -542,3 +542,21 @@ spec:
         - operation:
             ports:
               - "8022"
+---
```

Comment:
We don't need that, so you need to wait for 
https://github.com/openshift-knative/knative-istio-authz-chart/pull/11
to be merged, then re-run (or undo manually)